kotti_content_proxy/tests/test_models.py

```diff
@@ -1,18 +1,35 @@
 # -*- coding: utf-8 -*-
 
+import pytest
 
-def test_model(root, db_session, content):
 
+def test_model(foo_proxy):
+    assert foo_proxy[u'proxy'].proxied_object == foo_proxy[u'foo']
+
+
+def test_proxying_custom_attributes(root, db_session):
     from kotti.resources import Document
     from kotti_content_proxy.resources import ContentProxy
 
-    root[u'foo'] = Document(title=u'Foo')
+    foo = root[u'foo'] = \
+        Document(
+            title=u'Foo',
+            description=u'Bar',
+            body=u'This is the Foo document')
+    foo.some_attribute = u'some value'
     db_session.flush()
 
-    doc = root[u'foo']
-
-    # create a proxy for the document
-    root['proxy'] = ContentProxy(proxied_id=doc.id, title=u'proxied about')
+    # create a proxy for the 'foo'
+    proxy = root[u'proxy'] = \
+        ContentProxy(
+            proxied_id=foo.id,
+            proxied_attrs=['some_attribute', 'another_attribute'],
+            title=u'Proxy',
+            description=u'This is a proxy')
     db_session.flush()
 
-    assert root['proxy'].proxied_object == doc
+    assert proxy.some_attribute == foo.some_attribute
+    with pytest.raises(AttributeError) as excinfo:
+        proxy.another_attribute
+    assert excinfo.value.message == "'Document' object has no attribute " \
```

Comment:
Long strings are the one exception I make to the 80 chars rule.  Just append "  # noqa" to the end of the line to make it pass PEP8 checks.